internal/sim/generators.go

```diff
@@ -55,6 +150,148 @@ func Range(low, high int) Generator[int] {
 	})
 }
 
+// Strings
+
+// String returns a [Generator] that returns moderately sized readable strings,
+// with a bias towards smaller strings.
+func String() Generator[string] {
+	// Prefer smaller strings.
+	size := Weight([]Weighted[int]{
+		{100, Range(0, 10)},
+		{10, Range(10, 100)},
+		{1, Range(100, 4096)},
+	})
+	alphabet := OneOf(
```

Comment:
Do you want to throw in some non-English alphabet characters in there? Or English character-like runes that take up two bytes?

Reply:
Good idea. Added lowercase Greek letters, which are two bytes each in UTF-8.